pinot-minion/src/main/java/org/apache/pinot/minion/BaseMinionStarter.java

```diff
@@ -295,6 +296,7 @@ public String getStatusDescription() {
     });
 
     LOGGER.info("Pinot minion started");
+    _timezone = System.getProperty("user.timezone");
```

Comment:
if there is a conflict between system property and pinot config, what would you prefer?

Reply:
If it's only used by test, then I feel you can directly use `System.getProperty("user.timezone");` in the test?

Reply:
Thank you for the feedback. I will just remove this method entirely